specification/semantic_conventions.md

```diff
@@ -27,3 +28,41 @@ All Splunk distributions of OpenTelemetry,
 | Attribute  | Type | Description  | Examples  | Required |
 |---|---|---|---|---|
 | `splunk.distro.version` | string | The version number of the Splunk distribution being used. | `1.5.0` | Yes |
+
+Note: this section does not apply to Real User Monitoring libraries, as they do not use the OpenTelemetry Resource.
+
+## Real User Monitoring Spans and Attributes
+
+Real User Monitoring (RUM) libraries MUST set the `service.name` resource attribute to the value of the `applicationName`
+configuration property. This is the only resource attribute that RUM libraries are supposed to set because it's the only one
+the Zipkin exporter can understand.
+
+The following attributes MUST be added to all spans produced by RUM libraries:
+
+| Name                  | Type   | Description                                                                                                       |
+| ----                  | ----   | -----------                                                                                                       |
+| `app`                 | string | The value of the `applicationName` property configured in `SplunkRum`; same as `service.name` resource attribute. |
+| `splunk.rumSessionId` | string | The RUM session ID                                                                                                |
```

Comment:
It seems a bit odd this is camelCase and the version is period delimited? Should this be:

```suggestion
| `splunk.rum.sessionId` | string | The RUM session ID                                                                                                |
```

Reply:
Ideally it'd be `splunk.rum.session_id` (to match otel naming conventions), but it seems that this name is already hard-coded in the RUM backend and we can't really touch it.